cli_test/cli_posts_test.go

```diff
@@ -707,8 +707,8 @@ func TestDesmosCLIPostsReactions(t *testing.T) {
 	subspace := "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e"
 	reactions := types.Reactions{
 		types.Reaction{":earth:", "http://earth.jpg", subspace, fooAddr},
-		types.Reaction{":blush:", "https://gph.is/2p19Zai", subspace, fooAddr},
-		types.Reaction{":thumbsdown:", "https://gph.is/2phybnt", subspace, fooAddr},
+		types.Reaction{":plot:", "https://gph.is/2p19Zai", subspace, fooAddr},
```

Comment:
Can we revert all the changes that were made to this file?

Reply:
The changes I made were to fix some errors inside tests to exclude them from the possible causes that make the whole suite fail, should I revert them anyway?

Reply:
@bragaz Yes please, they were part of tests that have always been passing so I don't think they consitute any problem